scRNA-seq/04-dimension_reduction_scRNA.Rmd

```diff
@@ -85,6 +85,7 @@ For whatever reason, the way you read the data affects how it is stored in R.
 Reading from the directory results in smaller objects in R, so that is what we will do here. 
 
 Cell Ranger also outputs both filtered and raw matrices; today we will start with the raw matrix and perform our own filtering.
+![](diagrams/cellranger_1.png)
```

Comment:
I like the Cell Ranger diagram having a bit more detail w/ dimensionality reduction & clustering, but I would probably get rid of emptydrops, and just consider that rolled into filtering.

I'm also going to go through and make sure everything is aligned in the slides. My brain is very sensitive to some of the slight variations that I saw in the figures.

Reply:
I like the idea of including downstream analysis at the end and I will get rid of emptyDrops. I like the idea of changing to "expression quantification" too since this isn't really specific to cellranger. 

When I made these diagrams, I used the align tool in google slides to align them all to the middle, but I will double check to make sure all the diagrams are aligned.

Reply:
Yeah, the align tool on Google slides often doesn't align the part that it should, or that I think it should. I just moved the "expression quantification" to align with the other two line labels. Can you update the images once more? Sorry!

Reply:
yes should be all set now!